cli/src/node/mod.rs

```diff
@@ -119,9 +140,12 @@ impl Node {
 
     /// Initialize the node and return the init block id.
     pub async fn boot(&self, zerostates: Option<Vec<PathBuf>>) -> Result<BlockId> {
-        let boot_type = self
-            .overwrite_cold_boot_type
-            .unwrap_or(ColdBootType::LatestPersistent);
+        let boot_type = if !self.is_single_node {
+            self.overwrite_cold_boot_type
```

Comment:
Can we force update `overwrite_cold_boot_type` to Genesis with log about unused setting?

E.g.
```rust
if self.is_single_node {
    if let Some(boot_type) = self.overwrite_cold_boot_type 
        && boot_type != ColdBootType::Genesis 
    {
        tracing::warn!(?boot_type, "cold boot type settings was ignored due to single-node mode");
    }
    self.overwrite_cold_boot_type = Some(ColdBootType::Genesis);
}
```